plugins/janus_audiobridge.c

```diff
@@ -6993,6 +7132,14 @@ static void *janus_audiobridge_mixer_thread(void *data) {
 			g_list_free_full(anncs_list, (GDestroyNotify)janus_audiobridge_participant_unref);
 		}
 #endif
+		if(audiobridge->record == TRUE && audiobridge->wav_header_added == FALSE) {
```

Comment:
This can be simplified in

    if(audiobridge->record && !audiobridge->wav_header_added) {

Reply:
done